spec/features/sale_spec.rb

```diff
@@ -0,0 +1,264 @@
+      fill_in 'Mãe', with: 'Seller mother'
+      fill_in 'Pai', with: 'Seller father'
+      fill_in 'CEP', with: '64000-000'
+      fill_in 'Endereço', with: 'St. Seven, 34'
+      fill_in 'Bairro', with: 'Area test'
+      fill_in 'Complemento', with: 'Seller complement'
+      fill_in 'Cidade', with: 'Seller current city'
+      fill_in 'Estado', with: 'SS'
+      fill_in 'Telefone', with: '(86) 3232-3232'
+      fill_in 'Celular', with: '(86) 99900-1234'
+      fill_in 'E-mail', with: 'seller@test.com'
+
+      fill_in 'Preço de venda', with: 20000.0
+      fill_in 'Data da venda', with: '10/10/2016'
+    end
+
+    within '.purchaser-fields' do
+      fill_in 'Nome completo', with: 'Purchaser'
+      fill_in 'CPF', with: '000.000.000-01'
+      fill_in 'Data de nascimento', with: '10/10/1991'
+      fill_in 'RG', with: '1.234.568'
+      fill_in 'Emissor', with: 'SSP-PI'
+      fill_in 'Data de expedição', with: '10/10/2015'
+      fill_in 'Naturalidade', with: 'Purchaser City'
+      fill_in 'UF', with: 'CI'
+      fill_in 'Nacionalidade', with: 'Native'
+      fill_in 'Mãe', with: 'Purchaser mother'
+      fill_in 'Pai', with: 'Purchaser father'
+      fill_in 'CEP', with: '64000-001'
+      fill_in 'Endereço', with: 'St. Seven, 35'
+      fill_in 'Bairro', with: 'Purchaser Area'
+      fill_in 'Complemento', with: 'Purchaser complement'
+      fill_in 'Cidade', with: 'Purchaser current city'
+      fill_in 'Estado', with: 'PS'
+      fill_in 'Telefone', with: '(86) 3232-3233'
+      fill_in 'Celular', with: '(86) 99900-1233'
+      fill_in 'E-mail', with: 'purchaser@test.com'
+
+      fill_in 'Preço de compra', with: 22000.0
+      fill_in 'Data da compra', with: '20/10/2016'
+    end
+
+    within '.sale-fields' do
+      fill_in 'Número NFE', with: '123456'
+      fill_in 'NFE emitida em', with: '21/10/2016'
+      check 'Corretagem'
+      fill_in 'Observação', with: 'Observation about sale'
+    end
+
+    click_on 'Criar Venda'
+
+    expect(page).to have_content 'Venda criado com sucesso'
+
+    expect(page).to have_content 'Audi'
+    expect(page).to have_content 'A4 2.0 AT'
+    expect(page).to have_content 'Seller'
+    expect(page).to have_content 'Purchaser'
+    expect(page).to have_content '123456'
+    expect(page).to have_content '21/10/2016'
+    expect(page).to have_content 'Observation about sale'
+  end
+
+  scenario 'update' do
+    create(:sale,
+      seller: create(:customer, full_name: 'Seller'),
+      purchaser: create(:customer, full_name: 'Purchaser'),
+      vehicle: create(:vehicle,  brand: 'Audi', version: 'A4 2.0 AT', maker_year: 2014, model_year: 2015),
+      sale_price: 20000.0,
+      sale_date: '10/10/2016',
+      purchase_price: 22000.0,
+      purchase_date: '11/10/2016',
+      brokerage: false
+    )
+
+    click_on 'Vendas'
+    click_on 'Listar vendas'
+
+    expect(page).to have_content 'Listagem de vendas'
+
+    click_on 'editar'
+
+    expect(page).to have_content 'Editar veículo'
```

Comment:
You don't need to check this, because you will fill the form, so if the test fill the form it is on the form :P 

By the way if you are trying to check the content, I don't recommend you to check this kind of content, just because if you remove it it could never made your code break.